.circleci/config.yml

```diff
@@ -24,7 +27,7 @@ references:
 jobs:
   test:
     docker:
-      - image: "cimg/ruby:<< parameters.ruby_version >>"
+      - image: "cimg/ruby:<< parameters.ruby_version >>-browsers"
```

Comment:
I want to test the new flow with a system test that's javascript enabled, so I have to install chrome.